addons/hr/views/res_users.xml

```diff
@@ -91,7 +91,7 @@
                         </h1>
                     </div>
                     <div class="row">
-                        <h2 class="col-6">
+                        <h2 class="col-6 pl-0">
```

Comment:
Can you update your commit message like this:
```diff
- Currently, the profile view, the job title field is not aligned with
- other fields.
+ Before this commit, the job title field is not aligned with
+ other fields in the "My Profile" view.

- In this commit align the job title field with other fields in my progilr
+ This commit aligns the job title field with the other ones.
```
? Thanks in advance!

Reply:
@xavierbol , Done all the changes. Thanks